src/connectors/argentMobile/index.ts

```diff
@@ -242,4 +244,14 @@ export class ArgentMobileConnector extends Connector {
   }
 }
 
-export { isInArgentMobileAppBrowser } from "./helpers"
+export class ArgentMobileConnector {
+  static init(options: ArgentMobileConnectorOptions = {}) {
+    if (isInArgentMobileAppBrowser()) {
+      return new InjectedConnector({ options: { id: "argentX" } })
```

Comment:
shouldn't we also configure name and icon in the options here?

Reply:
yeah, will add tomorrow

Reply:
updated